examples/src/main/java/com/mapbox/navigation/examples/ui/NavigationViewFragment.kt

```diff
@@ -107,6 +107,7 @@ class NavigationViewFragment : Fragment(), OnNavigationReadyCallback, Navigation
                 optionsBuilder.directionsRoute(route)
                 optionsBuilder.shouldSimulateRoute(true)
                 optionsBuilder.enableVanishingRouteLine(true)
+                optionsBuilder.muteVoiceGuidance(true)
```

Comment:
https://github.com/mapbox/mapbox-navigation-android/pull/3289/files#r450393546

considering this example might be more suitable.